ui/initial_state.py

```diff
@@ -50,3 +53,9 @@ def write(state):
     write_state['shutting_down']     = False
     with open(state_file, 'w') as fh:
         pickle.dump(frozendict(write_state), fh)
+
+if __name__ == '__main__':
```

Comment:
See https://github.com/Bristol-Braille/canute-ui/pull/46#discussion_r95576527. If we fix our `__init__.py` it would be even cleaner.